test/acceptance/integration/notifications/mysql/mysql_column.feature

```diff
@@ -33,7 +33,7 @@ Feature: Stored in mysql new notifications per column from context broker
   Scenario Outline: stored new notifications in mysql from context broker with or without metadata
     Given copy properties.json file from "epg_properties.json" to test "mysql-sink" and sudo local "false"
     And configuration of cygnus instances with different ports "true", agents files quantity "1", id "test" and in "column" mode
-    And copy another configuration files and restart cygnus service and this execution is only once "true"
+    And copy flume-env.sh, matching table file from "matching_table.conf", log4j.properties, krb5.conf and restart cygnus service. This execution is only once "true"
     And verify if cygnus is installed correctly
     And "mysql" is installed correctly
     And create a new database "<tenant>"
```

Comment:
Yes, is possible to include mysql, mongo, fabric, remote-log, and another libraries... I append these libraries in iotpqautils